ui/packages/consul-ui/app/components/topology-metrics/index.hbs

```diff
@@ -20,7 +20,7 @@
   </div>
 {{/if}}
   <div id="metrics-container">
-    <div>
+    <div id="metrics-header">
```

Comment:
I know it doesn't necessarily matter right now, but I think we should avoid `id`s unless other methods are troublesome for some reason. It would be good to form a habit of reaching for classes (or anything but ids) over id's, and I'm pretty sure a `class` would be just as good here.